src/v2/auth.rs

```diff
@@ -133,14 +134,15 @@ impl WwwAuthenticateHeaderContent {
             .name("method")
             .ok_or(WwwHeaderParseError::FieldMethodMissing)?
             .as_str()
+            .to_lowercase()
             .to_string();
 
         let serialized_content = {
             let serialized_captures = captures
                 .iter()
                 .filter_map(|capture| {
                     match (
-                        capture.name("key").map(|n| n.as_str().to_string()),
+                        capture.name("key").map(|n| n.as_str().to_lowercase().to_string()),
```

Comment:
```suggestion
                        capture
                            .name("key")
                            .map(|n| n.as_str().to_lowercase().to_string()),
```
See `cargo fmt` from lints check